README.md

```diff
@@ -389,6 +394,9 @@ The `url` option may exclude the domain of the service URL since the domain
 must be equal to the URL defined in the LTI service anyway.
 The LTI parameters may be defined in the config.yaml file linked with
 the `config` option, thus it is not required to define them in RST.
+However if the config.yaml is defined in the submit directive, all lti-options
+in the submit directive are ignored and they must be written in the config.yaml
+instead.
```

Comment:
I think that this section must be made more clear about the two methods for defining LTI exercises: `submit` directive with the config option and using LTI options in the YAML file OR using LTI options in the submit directive with no config option. For example, using a list could make the alternatives more clear to the reader and differentiate them properly.